bin/gollum

```diff
@@ -151,7 +151,7 @@ MSG
   end
   opts.on("--user-icons [MODE]", [:gravatar, :identicon, :none], "Use specific user-icons for history view.",
     "Can be set to 'gravatar', 'identicon' or 'none'. Default: 'none'.") do |mode|
-    wiki_options[:user_icons] = mode
+    wiki_options[:user_icons] = mode.to_s
```

Comment:
This shouldn't be necessary, should it? I would assume that arguments are parsed as strings already?

Reply:
Annoyingly, `mode` is a symbol here. :(